### ISSUE TITLE
oasis.direct mobile mockup

### ISSUE THREAD

**Issue by pimato** [OasisDEX/oasis-direct#26]
Hey guys, 

I am finished up the first design mockups for oasis.direct on mobile.

**Starter Page**

![mobile-starter](https://user-images.githubusercontent.com/5337809/35860949-4dbdceea-0b46-11e8-9f7c-1626244cabc0.png)

**Finalize Page**

![mobile-finalize](https://user-images.githubusercontent.com/5337809/35861048-a9673f56-0b46-11e8-9322-0d0534be1354.png)

**Comment by peculiarity** [comment]
@pimato  Does those mockups mean that we remove the Landing ( Purple ) page and just display the app  on the whole screen  (taking all of the available space )?

**Comment by pimato** [comment]
@peculiarity I think oasis.direct has much more potential for beeing a useful and nice looking platform for lazy traders on decentralised exchanges. Besides providing a portal to liquidity from dexes, we could additionally add a trading interface where you can manage your trades.

**Comment by gbalabasquer** [comment]
@pimato I agree with @peculiarity that it is a bit strange the LP disappears in mobile as it is not even the brand there. At the same time I love how it looks, it looks actually as an application.
Maybe it could be added a purple bar on the top with the logo? In that case we would keep the application style but adding the same sense of the Desktop page.

**Comment by gbalabasquer** [comment]
What about the terms section? Where will they be described?

**Comment by peculiarity** [comment]
@gbalabasquer  we also have to really squeeze each and every pixel in height so that everything fits in there.

@pimato  The part with `I aceept Terms of service` , should this be a link and lead to somewhere else?

**Comment by arkadiuszsz** [comment]
@pimato i was thinking the same ( managing trades & trade alerts ).
I guess terms and conditions should show up in modal overlay when user taps the link.

**Comment by pimato** [comment]
@gbalabasquer, @peculiarity  I think we are going to create a disclaimer page so we are going to add a hyperlink on the word "Terms of Service". I don`t like popups more and more :/
